.github/workflows/ci.yml

```
@@ -360,14 +363,89 @@ jobs:
+  # docker build . --file ${{ env.DOCKERFILE_PHP_BETA }} --tag "php:$(date +%s)"
+  # docker run -d --name php_container "php:$(date +%s)"
   build-php-beta-manual:
     name: '[Manual] Build PHP beta'
     needs: manual
     runs-on: ubuntu-latest
+    environment: CI
     steps:
-    - uses: actions/checkout@v4
-    - name: php-fpm
-      run: docker build . --file ./php/beta.Dockerfile --tag "php:$(date +%s)"
+    - name: Check out the repo
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)